ext/standard/tests/general_functions/dl-001.phpt

```diff
@@ -0,0 +1,44 @@
+--TEST--
+dl(): Loaded extensions properly unregister their ini settings
+--SKIPIF--
+<?php
+
+if (extension_loaded('dl_test')) {
+    return;
+}
```

Comment:
Use
```
--EXTENSIONS--
dl_test
```
instead as since PHP 8.1 this is how the test runner loads extensions for Windows

Reply:
Won't this load the extension with `-dextension=`, though ? Here I really want the extension to be loaded with `dl()`

Reply:
Huuuuum IDK about that, @cmb69 how does this section work again?

Reply:
As of PHP 8.1.0, `--EXTENSIONS--` checks if the extension is already loaded, and otherwise uses `dl()` to load it (this is cross-platform behavior, not Windows specific). If `dl()` loading should be enforced, it's better not to use the `--EXTENSIONS--` section.

However, if a skip clause in `--SKIPIF--` fails, don't just return, but rather `die("skip …")` where "…" should state the reason for the skip; in this case maybe `die("skip dl_test extension already loaded")`.